hw/ip/otbn/data/otbn.hjson

```diff
@@ -162,9 +179,24 @@
       hwaccess: "hwo",
       hwext: "true",
       fields: [
-        { bits: "0",
-          name: "busy",
-          desc: "OTBN is performing an operation."
+        { bits: "7:0",
+          name: "status",
+          resval: 0,
+          desc: '''
+            Indicates the current state OTBN is in.
+
+            0 (IDLE): OTBN is not performing any action.
+
+            1 (BUSY_EXECUTE): OTBN is executing software.
+
+            2 (BUSY_SEC_WIPE_DMEM): OTBN is securely wiping the data memory.
+
+            3 (BUSY_SEC_WIPE_IMEM): OTBN is securely wiping the instruction
+                memory.
```

Comment:
Do these really need separating out?

Reply:
I can see wipe DMEM without wiping IMEM to be useful (want to keep the code in but for a new operation and want to avoid one run having any access to the data from the last) so I'd definitely retain the seperate wipe functionality. No objection to a new wipe both command.

Reply:
If the question is about separating out the BUSY enum specifically: Having them separate makes BUSY align with CMD, making it easier to reason about and specify.

It also helps debugging if OTBN gets stuck for whatever reason.


If the question is about wiping IMEM and DMEM separately: yes, that's a requirement (see also above).

Reply:
Ok, that makes sense.